packages/extras/src/lib/components/Billboard/Billboard.svelte

```diff
@@ -0,0 +1,50 @@
+<script lang="ts">
+	import { Group, Quaternion } from 'three';
+	import { T, useTask, useThrelte } from '@threlte/core';
+	import type { BillboardEvents, BillboardProps, BillboardSlots } from './Billboard.svelte';
+
+	type $$Props = Required<BillboardProps>;
+	type $$Events = BillboardEvents;
+	type $$Slots = BillboardSlots;
+
+	export let follow: $$Props['follow'] = true;
+	export let lockX: $$Props['lockX'] = false;
+	export let lockY: $$Props['lockY'] = false;
+	export let lockZ: $$Props['lockZ'] = false;
+
+	let inner: Group;
+	let localRef: Group;
+
+	const { camera } = useThrelte();
+
+	const q = new Quaternion();
+
+	useTask(() => {
+		if (!follow || !localRef) return;
```

Comment:
Instead of doing an early return we can avoid calling this function altogether by starting and stopping it with reactive statements:

```ts
const { start, stop } = useTask(() => {
  ...
}, { autoStart: false })

$: if (follow && localRef) {
  start()
} else {
  stop()
}
```

Reply:
Changed!

Reply:
That line can be removed! There should be no need to perform that check every frame anymore 😄